### DISCUSSION TITLE
Group by

### DISCUSSION THREAD

**Discussion by arogl** [TT-ReBORN/Georgia-ReBORN#193] [Q&A]
I was looking at making the playlist group by `%album artist%%date%%album%%track%` but that just ends up with the same group by for `%album artist%%original date%%album%%track%`.

Looking through the code I see date is defined in as date https://github.com/TT-ReBORN/Georgia-ReBORN/blob/56334e4ef35fc1966b754ea067a159688c2f8ab6/profile/georgia-reborn/scripts/Base/gr-config-defaults.js#L37-L57

To me it appears, the `%original date%` takes precedence. Is there a way to overwrite with the release date and year?

I am running a later git release (Oct 13 release)

**Comment by TT-ReBORN** [comment]
First of all, this is the wrong section and script file. 
These are all pre-defined defaults for the config file when its being written ( when the config does not exist ).
Open your `foobar2000\profile\georgia-reborn\configs\georgia-reborn-config.jsonc` file.

The `"title_format_strings"` section in the config file is for the Main UI.

If you want to change the Playlist grouping pattern, look in the same config file in the section `"themePlaylistGroupingPresets"`.
It is also good to read the headers of those section for more info:
```
// * PLAYLIST GROUPING PRESETS:
// * You can add new entries with grouping patterns or reorder entries that will be displayed in the Playlist grouping manager.
// * The playlist grouping manager can be accessed via right clicking in the Playlist > Grouping > Manage presets.
// * Changes will be saved to the config file after reload when creating new grouping presets or modifications in the Playlist grouping manager.
// * Note: These settings will NOT be automatically set if you use top menu Options > Settings > Theme configuration > Save settings to config file
```
The playlist grouping manager can be accessed via right clicking in the Playlist > Grouping > Manage presets.

-TT

**Reply by arogl** [reply]
I have already added the `georgia-reborn-config.jsonc` to my config under that section

```json
{ "name": "artist_year_album", "description": "by artist / year / album", "group_query": "%album artist%%date%%album%", "title_query": "[%album artist%]", "sub_title_query": "[%album%[ '('%albumsubtitle%')']][ - '['%edition%']']", "show_date": true, "show_disc": true },
```

I have also edited the `"title_format_strings"` section to have

```json
		"date": "%date%", // The full date stored for the track
```

I notice that the `sort by` also has a massive influence on this.

I think the issue was I saved the config and reloaded and re-grouped-by, but had the same effect. A restart, playing with sort by and group by seems to now have what I want of `album artist|date|album|track`

So much customisation you can do 😕

**Reply by TT-ReBORN** [reply]
I'm not sure what you wanted to do, but if it was only to sort by date there where already predefined presets in the "`Sort by`" menu 
in the playlist. You can sort the albums by ascending or descending. 

Anyways, all customization will be documented in the upcoming website.

-TT
